packages/atomic/src/components/facets/facet-search/facet-search.tsx

```diff
@@ -62,8 +62,8 @@ export class FacetSearch {
     }
 
     const search = regexEncode(query);
-    const regex = new RegExp(`(${search})`, 'ig');
-    return sanitize(resultValue).replace(regex, '<b>$1</b>');
+    const regex = new RegExp(`(${search})`, 'i');
+    return sanitize(resultValue).replace(regex, '<strong>$1</strong>');
```

Comment:
I'm not sure if `strong` is the correct html tag from an accessibility perspective because it has a semantic meaning of "this is important, needs your attention".

It's also is the inverse in this case, where the section wrapped by the tag actually has the normal font.

Maybe a `span` would be more suitable semantically speaking?

Ref: ["strong" vs. "bold"](https://www.geeksforgeeks.org/difference-between-strong-and-bold-tag-in-html/#:~:text=HTML%20bold%20tag%3A%20The%20bold,the%20formatting%20elements%20of%20HTML.&text=The%20main%20difference%20between%20these,text%20conventionally%20styled%20in%20bold.)